global/supportedLanguages.json

```diff
@@ -90,8 +90,7 @@
     "store": "Store",
     "submit": "Submit",
     "sun": "Sun",
-    "temperature": "Temperature (°C)",
-    "dateExceeds8Days": "*Cannot get weather for dates more that 8 days from now*",
```

Comment:
These changes shouldn't be removed, like we talked.

Reply:
I can still see these lines in this file